### PR TITLE
chore(meta): Updates roles and nomination processes

### PR DESCRIPTION
**Pull request by dipikabh**
This PR rewrites the role descriptions based on internal discussions.

TO DOs:

- [ ] "Processes" sections need to be redone based on actual steps
- [x] **Question**: 'Spotlight contributor' has been moved to 'Specialized roles' because it does not sit well with other 'Roles'. One idea is to change the title 'Specialized roles' to 'Special roles' so that it can include spotlight contributor, invited expert, and community manager better. (Update: done in https://github.com/mdn/content/pull/22916/commits/4cdd04fdd1d70db6ab0ec780f1f89522f9a92b31)
- [ ] **Question**: What info should be retained from [Volunteer and partner maintainers](https://developer.mozilla.org/en-US/docs/MDN/Community/Users_teams#volunteer_and_partner_maintainers)? Should we use the term 'partner maintainer' or is 'maintainer' good enough?
- [x] Add link to pull request guidelines if they are finalized before landing this (https://github.com/mdn/content/pull/22916/commits/8b9efccc051901303bf6a6599c49550d6aeb8c08)
- [ ] Add discord info for contacting the MDN team
- [x] [CONTRIBUTOR_ROLES.md](https://github.com/mdn/project-template/blob/main/CONTRIBUTOR_ROLES.md) to be removed: https://github.com/mdn/project-template/pull/21

AFTER THIS PR LANDS:
- [x] File name to be updated: users_teams --> roles_teams: https://github.com/mdn/content/pull/23534
- [x] Slug to be updated: MDN/Community/Users_teams --> MDN/Community/Roles_teams: https://github.com/mdn/content/pull/23534
- [ ] Sidebar to be updated; https://github.com/mdn/yari/pull/7943

### CODE REVIEW

File: files/en-us/mdn/community/users_teams/index.md [hunk 1]
```diff
@@ -10,12 +10,304 @@ tags:
...
+
+## Roles
+
+Irrespective of the role you take on in this project, you are always a [contributor](#contributor).
+A **contributor** is the base role and all other roles are built on top of it.
+You must satisfy the requirements of a contributor while working on this project in any capacity.
+
+### Contributor
+
+Contributors, or _community participants_, engage with the project and its community by contributing their time, skills, opinions, and ideas.
```
Comment (bsmth) [inline files/en-us/mdn/community/users_teams/index.md:30]: I've already read through most of this already, the only improvement I can think of is to simplify some of the language so it's not too wordy:

```suggestion
Contributors, or _community participants_, add to the project with their time, skills, opinions, and ideas.
```

File: files/en-us/mdn/community/users_teams/index.md [hunk 2]
```diff
@@ -10,12 +10,304 @@ tags:
...
+Contributors work on the project directly and add value to it.
+The criteria for what counts as a contribution is not limited to writing or testing code, but also includes creating and updating documentation, researching, fixing bugs, and helping other community members.
```
Comment (bsmth) [inline files/en-us/mdn/community/users_teams/index.md:32]: Simplify:

```suggestion
Aside from writing and testing code, contributions include creating and updating documentation, researching, fixing bugs, and helping other community members.
```

File: files/en-us/mdn/community/users_teams/index.md [hunk 3]
```diff
@@ -10,12 +10,304 @@ tags:
...
+The success and growth of the MDN Web Docs project is, in large part, due to our community of contributors. There are also employees, contractors, and a network of partners who are dedicated to the health, growth, and maintenance of the project as a whole. There are also some individuals who have committed a portion of their time to assist with the daily tasks of running MDN Web Docs. To manage all these different groups, the project relies heavily on [roles](#roles) and [teams](#teams) within our [GitHub organization](https://github.com/mdn). A list of the members of the organization can be [found here](https://github.com/orgs/mdn/people).
 
-A list of the members of the organization can be [found here](https://github.com/orgs/mdn/people).
+Community contributions help this open source project immensely.
+In return, contributors can use their work on this project to demonstrate their writing and technical skills as well as their ability to collaborate in team settings and engage with people from diverse backgrounds. This section describes the different roles you can assume while volunteering on the MDN Web Docs project.
```
Comment (bsmth) [inline files/en-us/mdn/community/users_teams/index.md:16]: Simplify maybe?

```suggestion
Contributors can use their work on MDN Web Docs to show writing, collaboration, technical skills, and the ability to work with people from diverse backgrounds. This document describes the roles you can take while volunteering on the MDN Web Docs project.
```

File: files/en-us/mdn/community/users_teams/index.md [hunk 4]
```diff
@@ -10,12 +10,304 @@ tags:
...
+
+The contributor roadmap is a step-by-step journey from one role to the next as you take on more responsibilities in the project.
+With the advancement in the contributor roadmap, you could serve more than one role at the same time.
+In some cases, such as with the [Invited experts](#invited_expert) role, it's possible to skip the stepwise progression and assume a role directly if you've demonstrated expertise in a particular area.
```
Comment (bsmth) [inline files/en-us/mdn/community/users_teams/index.md:20]: Maybe simplify

```suggestion
Roles such as [Invited experts](#invited_expert) can be directly obtained if you've demonstrated expertise in a particular area.
```

File: files/en-us/mdn/community/users_teams/index.md [hunk 5]
```diff
@@ -10,12 +10,304 @@ tags:
...
+- Recommend and vote for other members to become invited experts.
+- Attend weekly MDN Web Docs editorial call.
+
+### Community manager
+
+Community managers have a distinct role in many respects.
+Community managers share many of the same responsibilities as a [maintainer](#maintainer).
+In addition, community managers have the following responsibilities:
+
+- Addressing reports of violation of code of conduct and deciding on the appropriate action.
```
Comment (bsmth) [inline files/en-us/mdn/community/users_teams/index.md:198]: A link might be helpful

```suggestion
- Addressing reports of violation of [Mozilla's code of conduct](CODE_OF_CONDUCT.md) and deciding on the appropriate action.
```

File: files/en-us/mdn/community/users_teams/index.md [hunk 6]
```diff
@@ -10,12 +10,304 @@ tags:
...
+- At least three current Maintainers must approve the pull request.
+- Once the pull request is approved, the new Maintainer is added to the [Maintainers team](https://github.com/orgs/mdn/teams/maintainers) and any other relevant topic or project teams.
+-->
+
+### Nominating a spotlight contributor
+
+See who can be a [spotlight contributor](#spotlight_contributor).
+To nominate someone as a spotlight contributor, open an issue on GitHub:
+
+1. Go to [`Issues`](https://github.com/mdn/mdn/issues) in the `mdn/mdn` repository.
```
Comment (bsmth) [inline files/en-us/mdn/community/users_teams/index.md:239]: Same comment: https://github.com/mdn/mdn/issues/new/choose

File: files/en-us/mdn/community/users_teams/index.md [hunk 7]
```diff
@@ -10,12 +10,304 @@ tags:
...
+
+The MDN team will get in touch with the nominated person to get their information to be published on the [website](/en-US/) under "Contributor Spotlight".
+
+### Nominating an invited expert
+
+See who can be an [invited expert](#invited_expert).
+
+To nominate someone as an invited expert, open an issue on GitHub:
+
+1. Go to [`Issues`](https://github.com/mdn/mdn/issues) in the `mdn/mdn` repository.
```
Comment (bsmth) [inline files/en-us/mdn/community/users_teams/index.md:252]: Same 😄  https://github.com/mdn/mdn/issues/new/choose

File: files/en-us/mdn/community/users_teams/index.md [hunk 8]
```diff
@@ -10,12 +10,304 @@ tags:
...
+<!--
+TODO: Revisit this PR process
+
+- The nominator will open a pull request using the appropriate [template](https://github.com/mdn/mdn-community/roles/) against the [MDN Web Docs community repository](https://github.com/mdn/mdn-community/).
+- The nominee will add a comment to the pull request agreeing to all responsibilities of becoming an Invited Expert.
+- To be accepted, the pull request needs three approvals. This can be any combination of Invited Experts, or Maintainers.
+- Once the pull request is approved, the new Invited Expert is added to the [invited-experts team](https://github.com/orgs/mdn/teams/invited-experts) and the appropriate topic team.
+-->
+
+### Stepping down voluntarily
```
Comment (bsmth) [inline files/en-us/mdn/community/users_teams/index.md:266]: We're talking about voluntarily stepping down in the title, but the section is about inactivity. I would rename the section to something along the lines of commitment / what activity is.

For the section content, I thought the following, but depending on the title, you can throw this away:

```
Participants must be active to set an example and show commitment to the project.
Inactivity harms the project as it may lead to unexpected delays, contributor attrition, and a loss of trust in the project.

Inactivity is defined as:

- Not contributing to the project for at least six months.
- Not responding to communication for at least three months.

Inactivity results in a community role being revoked or downgraded and the member is moved to [emeritus status](#gaining-emeritus-status).
```

File: files/en-us/mdn/community/users_teams/index.md [hunk 9]
```diff
@@ -10,12 +10,304 @@ tags:
...
+
```
Comment (bsmth) [inline files/en-us/mdn/community/users_teams/index.md:267]: I believe we still need a "Stepping down" section where we state that it's okay to step away from roles due to any reason. We want participants to be able to let us know if they are overwhelmed, need a break, need to downgrade to a less-demanding role, or need more help, and which channels are best to talk to us etc.

File: files/en-us/mdn/community/users_teams/index.md [hunk 10]
```diff
@@ -35,7 +327,7 @@ A list of the members of the organization can be [found here](https://github.com
   - **Yari content SVG**: Responsible for reviewing docs under `/en-US/docs/Web/SVG`
```
Comment (bsmth) [inline files/en-us/mdn/community/users_teams/index.md:327]: Do we need to list all of these? Is it useful for this doc?

**Comment by bsmth** [comment]
Thanks Dipika, when we're nearly ready to merge, we should have a scan over the doc from a readability perspective, my first thoughts were to reduce the complexity of the language a little so it's less intimidating. 👍🏻

File: files/en-us/mdn/community/users_teams/index.md (hunk 1 above)
Comment (dipikabh) [reply files/en-us/mdn/community/users_teams/index.md:30]: Agree!!

File: files/en-us/mdn/community/users_teams/index.md (hunk 3 above)
Comment (dipikabh) [reply files/en-us/mdn/community/users_teams/index.md:16]: keeping "this section" because this document now contains a roles section and a teams section

File: files/en-us/mdn/community/users_teams/index.md (hunk 10 above)
Comment (dipikabh) [reply files/en-us/mdn/community/users_teams/index.md:327]: On one hand, we could remove the list and point to the mdn/teams link on GitHub. That will reduce the burden of updating this list here should any change happen to the teams (addition, removal).
On the other hand, explicitly listing the teams here might increase the likelihood of interested contributors to spot exactly the team they could help with for reviewing content (just an assumption).

Comment (dipikabh) [reply files/en-us/mdn/community/users_teams/index.md:320]: Valid feedback here 👍.

- I've merged the sections on "Stepping down voluntarily" and "Gaining emeritus status" under "Stepping down or applying for emeritus status"
- I've renamed "Getting demoted involuntarily" to "Demoting or removing inactive contributors" and reorganized the content in this section

Let me know what you think

File: files/en-us/mdn/community/users_teams/index.md (hunk 9 above)
Comment (dipikabh) [reply files/en-us/mdn/community/users_teams/index.md:264]: Valid feedback here 👍.

- I've merged the sections on "Stepping down voluntarily" and "Gaining emeritus status" under "Stepping down or applying for emeritus status"
- I've renamed "Getting demoted involuntarily" to "Demoting or removing inactive contributors" and reorganized the content in this section

Let me know what you think

**Comment by dipikabh** [comment]
In https://github.com/mdn/content/pull/22916/commits/4cdd04fdd1d70db6ab0ec780f1f89522f9a92b31:
- changed title from "Contributor roles and teams" to "Roles and teams on MDN"
- added a table to compare and summarize the roles (section "Summary of the roles")
- changed the section heading "Specialized roles" to "Special roles"

File: files/en-us/mdn/community/users_teams/index.md (hunk 9 above)
Comment (bsmth) [reply files/en-us/mdn/community/users_teams/index.md:274]: Looks good 👍🏻

File: files/en-us/mdn/community/users_teams/index.md [hunk 11]
```diff
@@ -10,12 +10,307 @@ tags:
...
+
+Owners can:
+
+- Add and remove organization owners and members as needed.
+- Add and remove collaborators to specific repositories as needed.
+- Add repositories (as fresh projects or transfers) as needed.
+
+### Summary of the roles
+
+||[Contributor](#contributor)|[Organization member](#organization_member)|[Maintainer](#maintainer)|[Owner](#owner)|
```
Comment (bsmth) [inline files/en-us/mdn/community/users_teams/index.md:153]: What do you think about flipping the orientation to:

| Role                    | Requirements                                                                                                                                                                                                                                                                | Privileges                                                                                               |
| ----------------------- | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | -------------------------------------------------------------------------------------------------------- |
| **Contributor**         | Follow the code of conduct and contribution guidelines                                                                                                                                                                                                                      | - Invitations to contributor events<br>- Eligibility to become an organization member                    |
| **Organization member** | - Enable 2FA for GitHub account<br>- Enable signed commits<br><br>One or more of:<br>- Resolve two or more issues<br>- Contribute for at least two months<br>- Active contribution in a project area                                                                        | Access rights at organization level                                                                      |
| **Maintainer**          | One or more of:<br>- Invited expert for at least six months<br>- Knowledge across multiple project areas<br>- Act towards overall health of the project<br>- Mentor other contributors<br>- Spend at least 16 hours per month on the project<br>- Attend community meetings | Approve and merge pull requests                                                                          |
| **Owner**               | Limited to Mozilla staff                                                                                                                                                                                                                                                    | - Manage access of different roles to various repositories<br>- Add or archive repositories and projects |

Comment (dipikabh) [reply files/en-us/mdn/community/users_teams/index.md]: Makes better sense this way, thanks! Fixed in https://github.com/mdn/content/pull/22916/commits/4c82f8a3a02a5c1ed6747140fea98a83db67984d

File: files/en-us/mdn/community/users_teams/index.md (hunk 10 above)
Comment (dipikabh) [reply files/en-us/mdn/community/users_teams/index.md]: Thanks for the idea! Shortened the list and also updated team names as per https://github.com/orgs/mdn/teams. Fix in https://github.com/mdn/content/pull/22916/commits/565842e3c057d25994bc44cd2e71618c5cf466da.

**Comment by dipikabh** [comment]
Thanks a lot @bsmth for reviewing this page. Let me know if there's anything else that can be improved.
There are a few pending [TO DOs](https://github.com/mdn/content/pull/22916#issue-1493356007). I can take them up in a subsequent PR.

File: files/en-us/mdn/community/users_teams/index.md (hunk 10 above)
Comment (bsmth) [reply files/en-us/mdn/community/users_teams/index.md]: Looks good!

File: files/en-us/mdn/community/users_teams/index.md [hunk 12]
```diff
@@ -10,113 +10,334 @@ tags:
...
+The success and growth of the MDN Web Docs project is, in large part, due to our community of contributors. There are also employees, contractors, and a network of partners who are dedicated to the health, growth, and maintenance of the project as a whole. There are also some individuals who have committed a portion of their time to assist with the daily tasks of running MDN Web Docs. To manage all these different groups, the project relies heavily on [roles](#roles) and [teams](#teams) in the [MDN organization on GitHub](https://github.com/mdn). A list of the members of the organization can be [found here](https://github.com/orgs/mdn/people).
```
Comment (bsmth) [inline files/en-us/mdn/community/users_teams/index.md:13]: We have 2x "There are also" sentences here, maybe we want to fix it in this PR.

Comment (bsmth) [inline files/en-us/mdn/community/users_teams/index.md:13]: Maybe like this?

```suggestion
The success and growth of the MDN Web Docs project is, in large part, due to our community of contributors. Changes to the site are performed by employees, contractors, and a network of partners dedicated to the health, growth, and maintenance of the project. Some individuals have committed a portion of their time to assist with running MDN Web Docs. The project relies heavily on [roles](#roles) and [teams](#teams) in the [MDN organization on GitHub](https://github.com/mdn) to manage and incorporate changes from these different groups. A list of the organization's members can be [found here](https://github.com/orgs/mdn/people).
```

File: files/en-us/mdn/community/users_teams/index.md [hunk 13]
```diff
@@ -10,113 +10,334 @@ tags:
...
+
+- **Core**: Core MDN Web Docs team
+- **mdn-community-engagement**: People responsible for community engagement across our repositories
+- **mdn-product**: People responsible for the MDN Plus product
+- **localization-team-leads**: People who lead our individual localization teams
+- **OWD**: Contributors from the Open Web Docs non-profit organization
+- **sre**: Site reliability engineers who support MDN Web Docs
+- **yari-content**: The umbrella team for all MDN Web Docs content reviewers
+
+  - There is a sub-team for the different topic areas — accessibility, Add-ons, CSS, HTML, HTTP, Javascript, SVG, Web API, and Web assembly. For example, there's yari-content-css and yari-content-svg.
```
Comment (bsmth) [inline files/en-us/mdn/community/users_teams/index.md:324]:
```suggestion
  - There is a sub-team for the different topic areas — accessibility, Add-ons, CSS, HTML, HTTP, Javascript, SVG, Web API, and Web assembly. For example, there's `yari-content-css` and `yari-content-svg`.
```

We could also do `@yari-content-css` variant as I guess that's how they're tagged on GitHub

File: files/en-us/mdn/community/users_teams/index.md [hunk 14]
```diff
@@ -10,113 +10,334 @@ tags:
...
+  - There are also sub-teams for different languages — Brazilian Portuguese, Chinese, French, Japanese, Korean, Russian, and Spanish. For example, there's yari-content-fr and yari-content-ko.
```
Comment (bsmth) [inline files/en-us/mdn/community/users_teams/index.md:325]: Same comment here

**Comment by bsmth** [comment]
Thanks @dipikabh, I had another read through and found some quick edits we can do, the rest we can look at after it's merged. Thanks for the overhaul on this 🙌🏻

File: files/en-us/mdn/community/users_teams/index.md [hunk 15]
```diff
@@ -1,5 +1,5 @@
 ---
-title: Owners and teams
+title: Roles and teams on MDN
```
[the remark this answers is not in the document]
Comment (dipikabh) [reply files/en-us/mdn/community/users_teams/index.md:2]: Keeping the order as roles first and then teams though to match the order of content.

File: files/en-us/mdn/community/users_teams/index.md (hunk 12 above)
Comment (dipikabh) [reply files/en-us/mdn/community/users_teams/index.md]: Done in https://github.com/mdn/content/pull/22916/commits/d34e7983411723fb927a843269909b2e2900cb76. Thanks for pointing it out.

Comment (dipikabh) [reply files/en-us/mdn/community/users_teams/index.md]: Oops, saw this after the previous commit. I've incorporated some of these edits. Thanks for suggestions!

**Review by bsmth — APPROVED** [review]
Thanks, Dipika! 🙌🏻

File: files/en-us/mdn/community/users_teams/index.md (hunk 13 above)
Comment (dipikabh) [reply files/en-us/mdn/community/users_teams/index.md]: That's a good suggestion. I added the '@' to all other team names and removed the bold formatting https://github.com/mdn/content/pull/22916/commits/fd586853a57a99b8d8afa2484cda7799f57f1c6d
